src/int64_util.js

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
I copied this file as is from thrift repo under Apache 2.0 license

Reply:
Can't we just use the other package rather than copying it here?

As far as I can tell, copying it here is fine as long as we preserve the license and acknowledge the source:

https://opensource.stackexchange.com/questions/1713/adding-feature-to-an-apache-v2-package

Yet I'd prefer to use it as is (unless there is an impediment) - if we are going to keep this code here, let's make sure it's properly linted and updated to our standards.

Reply:
@thehcma I think it should be possible to build a library that can work with different types. So instead of having json-long and json-int64, we can created one library that is configurable to work with different types.